spec/ant_spec.lua

```diff
@@ -82,6 +82,15 @@ describe("Arweave Name Token", function()
 		assert.are.same(_G.Records["@"].ttlSeconds, 900)
 	end)
 
+	it("gets all records", function()
+		records.setRecord("zed", string.rep("1", 43), 3600)
+		records.setRecord("@", string.rep("1", 43), 3600)
```

Comment:
nit - generally when testing you should avoid "using the tools your testing" - instead, set the records manually in global state using `_G.Records = {}` and _then_ validate the behavior

Reply:
good point - https://github.com/ar-io/ar-io-ant-process/pull/35/commits/16220a98103d5b7c913d7c304baeebfa4b66ea9d